routes/kiosk.php

```diff
@@ -3,4 +3,10 @@
 use Illuminate\Support\Facades\Route;
 use App\Http\Controllers\InteractiveFeatureExperiencesController;
 
-Route::get('/interactive-features/{slug}', [InteractiveFeatureExperiencesController::class, 'showKiosk'])->name('interactiveFeatures.showKiosk');
+Route::get(
+    '/interactive-features/{slug}',
+    [InteractiveFeatureExperiencesController::class, 'showKiosk']
+)->name('interactiveFeatures.showKiosk');
+
+// Only needed so that the kiosk doesn't fallback to the web routes.
+Route::fallback(fn() => abort(404));
```

Comment:
This prevents the router from trying to look up a generic page by slug from the `web.php` routes:
https://github.com/art-institute-of-chicago/artic.edu/blob/develop/routes/web.php#L231